.gitignore

```diff
@@ -1,2 +1,6 @@
 demo-dist
 node_modules
+*.log
```

Comment:
*.log is to exclude the npm-debug.log generated by npm when npm install fails (among other things)

.idea is to exclude the project setting folder automatically created by the PHPStorm/WebStorm IDE which I'm using

benchmark is to exclude the folder with my benchmark (and the 255ko obj test file). I didn't want to serve those files to the end users and another way to avoid it would have been to introduce some additional complexity in the repo by adding a .npmignore or use the `files` field in the package.json

Those changes are not essential by any means